### DISCUSSION TITLE
DPL regelt auch Persistant Limit

### DISCUSSION THREAD

**Discussion by LinuxUnix80** [hoylabs/OpenDTU-OnBattery#1418] [Q&A] [answered]
Hallo,

ich habe meine beiden 4MB ESP32 DTUs gegen ESP32-S3 16MB ausgewechselt, mit der aktuellsten OpenDTU on Battery Version.
Nun ist mir aufgefallen, dass wenn ich DPL einschalte, er nicht nur das NON_Persistant Limit regelt, sondern auch das PERSISTANT Limit. Ist das so gewollt? Dachte um den WR zu schonen, wird nur der NON Bereich geregelt?

OpenDTU on Battery Version: 2024.11.20
WR: HMS-2000-4T
DTU HW: ESP32-S3, CMT2300A Chip

**Comment by spcqike** [comment, marked as answer]
https://github.com/hoylabs/OpenDTU-OnBattery/discussions/1093

**Reply by LinuxUnix80** [reply]
Hi Danke für den Hinweis. Warum zeigt er mir das dann an:
![grafik](https://github.com/user-attachments/assets/869d5b79-453c-400e-95e0-877c631a6c33)


Müsste nicht (bei einem  800W WR, bei Persistant 800W stehenbleiben und nur NON sich ändern?)Habe ich da einen Denkfehler?

**Reply by spcqike** [reply]
https://github.com/hoylabs/OpenDTU-OnBattery/discussions/1093#discussioncomment-10009790

> Die Ursache wird die gleiche sein: Diese Topics (oder besser: Sensoren) gibt es jeweils getrennt voneinander, weil sie beim Stellen verschiedene Topics beschreiben, sodass sie verschiedene Aktionen auslösen. Aber die Quelle für die Anzeige des Wertes ist jeweils nur das aktuell eingestellt Limit, egal ob das zuletzt eingestellt Limit persistent oder flüchtig gesetzt wurde. Das muss auch so sein, dann der Wechselrichter meldet nur ein Limit zurück, nämlich das aktuelle, egal ob das mit Persistenz eingestellt wurde, oder nicht.

**Reply by Manos1966** [reply]
@LinuxUnix80  I hoffe du hast nicht dagegen, dein Home Assistant Bild uns zur Verfügung zu stellen:

https://github.com/hoylabs/OpenDTU-OnBattery/wiki/Home-Assistant#home-assistant-shows-opendtu-onbattery-changing-the-non-persistant-as-well-as-the-persistant-limit-of-my-inverter

**Reply by LinuxUnix80** [reply]
> @LinuxUnix80 I hoffe du hast nicht dagegen, dein Home Assistant Bild uns zur Verfügung zu stellen:
> 
> https://github.com/hoylabs/OpenDTU-OnBattery/wiki/Home-Assistant#home-assistant-shows-opendtu-onbattery-changing-the-non-persistant-as-well-as-the-persistant-limit-of-my-inverter

Moin,

das ist in Ordnung! Hatte diesen Fakt wohl vergessen gehabt :) Danke für die Antworten und den Wiki-Eintrag.
